include/halo/utils/passes_helper.h

```diff
@@ -0,0 +1,169 @@
+//===- passes_helper.h ------------------------------------------*- C++ -*-===//
+//
+// Copyright (C) 2019-2021 Alibaba Group Holding Limited.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+#ifndef HALO_UTILS_PASSES_HELPER_H_
+#define HALO_UTILS_PASSES_HELPER_H_
+
+#include <iostream>
+
+#include "halo/lib/framework/common.h"
+#include "halo/lib/parser/parser.h"
+#include "halo/lib/pass/pass_manager.h"
+#include "halo/lib/target/generic_llvmir/generic_llvmir_codegen.h"
+#include "halo/lib/transforms/fusion.h"
+#include "halo/lib/transforms/reorder_channel.h"
+#include "llvm/ADT/Triple.h"
+
+namespace halo {
+__attribute__((unused)) static void PopulateCodeGenPasses(
```

Comment:
we can define a MACRO like `#define HL_UNUSED __attribute__((unused)) `

Reply:
Good idea.